index.html

```diff
@@ -2,9 +2,174 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Een blok positioneren</title>
+    <link rel="stylesheet" href="./styles.css">
 </head>
 <body>
+<header class="header__outer-container">
+    <div class="header__inner-container">
+        <nav class="header__navbar">
+            <ul class="header__navbar-list">
+                <li class="header__navbar-item"><a href="./pages/not found">case studies</a></li>
```

Comment:
Hier heb je de list items allemaal een class gegeven, dit is enigszins wat overbodige luxe omdat we altijd maar 1 navigatie hebben. We kunnen dus de ul en de a's aanspreken vanuit een descendant selector :)